pkgs/tools/misc/ets/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ets";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "zmwangx";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XWIDo5msTMTLr60x1R9cwsiZIDG6G+uHWx8idt4F2iA=";
+  };
+
+  patches = [ ./go-mod.patch ];
+
+  vendorSha256 = "sha256-+8dXfqOu8XTw2uEx3GAynQSHtzifejZtddr1CdxrupA=";
+
+  ldflags = [ "-s" "-w" "-X main.version=v${version}-nixpkgs" ];
+
+  preBuild = ''
+    rm -rf fixtures
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share/man/man1
+    cp ets.1 $out/share/man/man1
```

Comment:
```suggestion
    installManPage ets.1
```

Reply:
Thanks, I figured there was something like this but didn't find it.